src/main/java/com/blackducksoftware/integration/hub/alert/datasource/relation/HubUserProjectVersionsRelationPK.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (C) 2017 Black Duck Software, Inc.
+ * http://www.blackducksoftware.com/
+ *
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.blackducksoftware.integration.hub.alert.datasource.relation;
+
+import java.io.Serializable;
+
+public class HubUserProjectVersionsRelationPK implements Serializable {
+    private static final long serialVersionUID = 8206061503103229279L;
+
+    public Long userConfigId;
+    public String projectName;
+    public String projectVersionName;
+
```

Comment:
This class functions as a primary key object for database operations. For serialization purposes, only a default constructor is recognized.